bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -533,6 +537,23 @@ public void channelUnlinked(ChannelUID channelUID) {
         logger.debug("channel {} unlinked ", channelUID.getAsString());
     }
 
+    public InsteonAddress getInsteonAddress() {
+        if (config != null) {
+            return new InsteonAddress(config.getAddress());
+        } else {
+            throw new IllegalArgumentException("config is null");
+        }
+    }
+
+    public void deviceNotLinked() {
+        String address = config != null ? config.getAddress() : "unknown";
```

Comment:
I suppose address could be configured as "unknown", so this is kind of a strange message (_"device with the address 'unknown' was not found"_). See previous comment as this might also solve this one.

Reply:
changed